pkg/binding/bindingpolicy-resolution-broker.go

```diff
@@ -44,6 +45,7 @@ type ResolutionBroker interface {
 // Resolution is a struct that represents the resolution of a binding policy.
 // It contains the destinations and object data for the resolution.
 type Resolution struct {
+	Name string
```

Comment:
Note to self: why is the name needed here, doesn't the user already know the name?